source/plugins/go/src/network_flow_logs.go

```diff
@@ -286,12 +294,28 @@ func mapNetworkFlowLogsToDataMap(dataMap map[string]interface{}, record map[stri
 	if traceObservationPoint := extractString(flow, "trace_observation_point"); traceObservationPoint != "" {
 		dataMap["TraceObservationPoint"] = traceObservationPoint
 	}
-	// Packets and Bytes
-	if packetsSent, ok := flow["packets_sent"]; ok {
-		dataMap["PacketsSent"] = safeToInt(packetsSent)
-	}
-	if packetsReceived, ok := flow["packets_received"]; ok {
-		dataMap["PacketsReceived"] = safeToInt(packetsReceived)
+	// Flow counts and packets
+	if MdsdNetworkFlowLogsStreamTagName == getOutputStreamIdTag(ContainerNetworkLogsStreamName, "", &NetworkFlowTagRefreshTracker) {
```

Comment:
instead of this, you can just use boolean flag based on stream obtained in https://github.com/microsoft/Docker-Provider/blob/1f251a8ae3b3a40a51567fb271b9256da8e3a903/source/plugins/go/src/network_flow_logs.go#L82 , right?

Reply:
from acns: "PacketsSent & PacketsReceived were never mapped to anything in the flow object. 
So now instead of them we have the new fields which still will not be mapped. So really only the names of columns are changing.
For AdditionalData in Extensions is it possible to keep that but exclude the three new columns? 
Extensions can contain other fields."
 so removed this check as those fields were not there before